software/playground.py

```diff
@@ -11,4 +11,46 @@
 
 bb.measure()
 print(bb.cellVolts)
-print(bb.cellTemps)
\ No newline at end of file
+print(bb.cellTemps)
+
+#blame Lev below this line
+from datetime import datetime
+import csv
+
+def write_to_csv(date, voltage):
+"""
+function takes in timestamp and voltage, 
+writes values to csv file, 
+"a" appends whereas "w" clears file first
+"""
```

Comment:
Should be tabbed over to same indent as function code

Reply:
Also good to know.